CMS-Enterprise/mint-app
# [MINT-3226] Edit Solutions Linked to Milestones

# MINT-3226

## Description


Added support for optionally editing milestones to set linked solutions and an endpoint for directly setting linked solutions

## How to test this change

- Postman
  - Create a new MTO Milestone
  - Create a new MTO Solution
  - Edit the milestone, passing one or more linked solution IDs and common solution keys
  - Validate expected results
  - Experiment with removing linked solution IDs / common solution keys and validate the results
  - Use the set milestone linked solutions endpoint
  - Experiment with inputs and validate results


## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
